docs/src/ecadd/spec.md

```diff
@@ -0,0 +1,107 @@
+---
```

Comment:
For this to work as expected I think you'll need a new line before the last `---` and after the first `---`. Otherwise it works like the title of the whole page.